final_project/snapvisite/migrations/0010_alter_schedule_close_time_alter_schedule_open_time.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.4 on 2022-05-08 19:57
+
+import datetime
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('snapvisite', '0009_alter_schedule_close_time_alter_schedule_day_of_week_and_more'),
+    ]
+
```

Comment:
Jak mamy jeden pull request to powinna być jedna migracja na apkę dodawana, nikt nie musi po migracjach wiedzieć, że coś tam dodawałeś, potem usuwałeś, potem znowu dodawałeś, chcemy znać tylko co zostało zmienione w modelach po tym pull requeście